src/neuroglancer/annotation/type_handler.ts

```diff
@@ -163,6 +163,8 @@ interface AnnotationTypeRenderHandler<T extends Annotation> {
   };
   sliceViewRenderHelper: {new(gl: GL): AnnotationRenderHelper;};
   pickIdsPerInstance: number;
+  getRepresentativePoint: (objectToData: mat4, data: ArrayBuffer, offset: number, partIndex: number, annotation: T) => vec3;
```

Comment:
Do we need the data and offset parameters?  All of the information should be present in the annotation object.